.github/workflows/test1.yml

```diff
@@ -11,6 +11,10 @@ on:
       - 'metrics/**'
       - 'docs/**'
       - '**.md'
+
+      # Should run the workflow if changes are made to the following files
+      - '!waku/v1/**.nim'
+      - '!tests/v1/**.nim'
```

Comment:
This was implicit. Please remove this. For example, the "whisper" protocol changes are not included here.

Reply:
> I don't see the increment compared to the current CI workflow definitions. Can you clarify the following statement? What are you trying to solve with it?
> 
> > fix(ci): when docs and code is changed, workflows run
> 
> Additionally, when you change the `ignore-paths` section in `test2.yaml`, you have to do the complementary with `test2-ingore.yml` otherwise, the checks will remain not executed since there will be files that are not covered by any of the workflows.

Refer to this PR - https://github.com/status-im/nwaku/pull/1386. Tests are skipped even though there are changes made to the code.